README.md

```diff
@@ -3,6 +3,11 @@ Place to share presentation material from the monthly Web Developer Meetup in Ch
 
 ## 2019
 
+### July
```

Comment:
```suggestion
### July
#### Event Sourcing
```

Reply:
@MorrisJobke perfect!
Unfortunately only @ronnyhartenstein has the right to commit this suggestion to his branch.

Reply:
He also has the option to grant write permission for this branch when the PR was opened (there is a little checkbox) ... just for the future. 😉

Reply:
Waah, sorry, I forgot about that. Hold on a minute..